package.json

```diff
@@ -211,7 +211,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/formation": "^1.3.0",
+    "@department-of-veterans-affairs/formation": "^1.3.1",
```

Comment:
Pagination uses anchor tags without hrefs which are not added to the tab order by default. Opted for a short term fix in the design system to add this functionality and made a ticket to switch the anchor tags to buttons.